scripts/inc.installer.sh

```diff
@@ -713,16 +713,19 @@ nogooglewebview_removal_msg="NOTE: The Stock/AOSP WebView is not available on yo
 if [ -n "$(cat /proc/cmdline | grep slot_suffix)" ];
 then
   device_abpartition=true
-  SYSTEM=/system/system
+  SYSTEM_MOUNT=/system
+  SYSTEM=$SYSTEM_MOUNT/system
   VENDOR=/vendor/vendor
-elif [ -n "$(cat /proc/mounts | grep /system_root)" ];
+elif [ -n "$(cat /etc/fstab | grep /system_root)" ];
```

Comment:
Why isn't the mount available in proc? that should be more reliable than a fstab (because it checks if the kernel has actually mounted this location

Reply:
It's because /proc/mounts shows only the partitions currently mounted, so if system isn't, only fstab will contain the path
So it's better to use /etc/fstab because /proc/mounts just doesn't work

Reply:
Alright, I understand!